docs/source-pytorch/debug/debugging_intermediate.rst

```diff
@@ -42,16 +42,26 @@ in the training set to turn off shuffle for you.
 ********************************
 Look-out for exploding gradients
 ********************************
-One major problem that plagues models is exploding gradients. Gradient norm is one technique that can help keep gradients from exploding.
+One major problem that plagues models is exploding gradients.
+Gradient clipping is one technique that can help keep gradients from exploding.
+
+You can keep an eye on the gradient norm by logging it in your LightningModule:
 
 .. testcode::
 
-    # the 2-norm
-    trainer = Trainer(track_grad_norm=2)
+    from lightning.pytorch.utilities import grad_norm
```

Comment:
The PyTorch docs CI job fails here because this triggers the lightning import but `fastapi` is not installed.